MovieSharing/VideoModel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Video.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 28.09.21.
+//
+
+import Foundation
+
+struct VideosResponse:Decodable {
+    
+    var items: [Video]
+    
+}
+
+struct Video:Decodable {
```

Comment:
it would be good to create a new file for every struct/class that is named like the struct/class